src/mpyl/schema/run_properties.schema.yml

```diff
@@ -44,7 +44,7 @@ properties:
             type: string
           user_email:
             description: Email of of the user that triggered the run
-            type: ["string", "null"]
+            type: string
```

Comment:
Actually null, is good here. But it only _really_ works when this change is merged https://github.com/Vandebron/mpyl/pull/67/commits/0a56741d628df881e5b02df48306a3319fc5362c